app/src/main/java/com/vasilevkin/catsanddogs/features/splash/presenter/SplashPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.vasilevkin.catsanddogs.features.splash.presenter
+
+import com.vasilevkin.catsanddogs.features.splash.ISplashContract
+
+
+class SplashPresenter(private var view: ISplashContract.View?) : ISplashContract.Presenter {
```

Comment:
you have two views - main and splash, try to avoid DRY through baseFragment, basePresenter

Reply:
fixed.